src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -161,6 +161,11 @@ - (instancetype)initWithHost:(NSString *)host
     [NSException raise:NSInvalidArgumentException
                 format:@"The requests writer can't be already started."];
   }
+  NSString *check_host = [NSURL URLWithString:[@"https://" stringByAppendingString:host]].host;
```

Comment:
If we put this here, would it still cause hostname with zone id to be failed?

Reply:
yeah sorry. Not sure what I was thinking.